tests/mathml_nemeth/general_001.test/Makefile.am

```diff
@@ -0,0 +1,3 @@
+EXTRA_DIST =		\
+	expected.txt	\
+	input.xml
```

Comment:
If you want the png to be in the distribution tarball you should mention it here in `EXTRA_DIST`.